protocol/0044-LIME-lp_mechanics.md

```diff
@@ -115,38 +141,58 @@ The [liquidity_fee](./0029-FEES-fees.md) of a market on Vega takes as an input,
 
-### Checking margins for orders
+Committed Liqudity Providers are required to provide a multiple of their stake (supplied in the settlement currency of the market) in notional volume of orders within the range defined below.
 
-As pegged orders are parked during an auction and not placed on the book, margin checks will not occur for these orders. This includes checking the orders margin when checking the validity of the transaction so orders are accepted. Open positions are treated the same as any other open positions and their liquidity provider orders are pegged orders and will be treated the same as any other pegged orders.
+The multiple is controlled by a network parameter `market.liquidity.stakeToCcyVolume`:
+```
+liquidity_required = stake_amount ✖️ market.liquidity.stakeToCcyVolume
+```
 
-## Liquidity provision and penalties
+### Meeting the committed volume of notional 
 
-### Calculating liquidity from commitment
+#### During continuous trading
+If there is no mid price then no LP is meeting their committed volume of notional. 
+If there is mid price then we calculate the volume of notional that is in the range
+```
+(1.0-market.liquidity.priceRange) x mid =< price levels that count <= (1+market.liquidity.priceRange)x mid.
+```
+If this is greater than or equal to `liquidity_required` then the LP is meeting the committed volume of notional.
 
-Each liquidity provider supplies an amount of liquidity which is calculated from their commitment (stake) and measured in 'currency siskas' (i.e. USD siskas, ETH siskas, etc.).This is calculated by multiplying the stake by the network parameter `market.liquidity.stakeToCcyVolume` as follows:
+#### During auctions 
+We calculate the volume of notional that is in the range
+```
+(1.0-market.liquidity.priceRange) x min(last trade price, indicative uncrossing price) =<  price levels that count <= (1.0+market.liquidity.priceRange) x max(last trade price, indicative uncrossing price). 
+```
+If this is greater than or equal to `liquidity_required` then the LP is meeting the committed volume of notional.
 
-`lp_liquidity_obligation_in_ccy_volume = market.liquidity.stakeToCcyVolume ⨉ stake`.
 
-Note here "ccy" stands for "currency". Liquidity measure units are 'currency x volume'. This is because the calculation is basically `volume ⨉ price of the volume` and the price of the volume is in the said currency.
+Note: we don't evaluate whether LPs meet the SLA during opening auctions so there will always be a mark price. 
 
-### How liquidity is supplied
 
-When a liquidity provider commits to a market, the LP Commitment transaction includes a _buy shape_ and _sell shape_ which allow the LP to spread their liquidity provision over a number of pegged orders at varying distances from the best prices on each side of the book. These 'shapes' are used to generate pegged orders (see the [liquidity provision order type spec](./0038-OLIQ-liquidity_provision_order_type.md)).
+### Penalty for not meeting the SLA 
```

Comment:
Good point!

Three options:

1. Unforced (i.e. not by bond confiscation) commitment reductions only take effect at epoch end. New commitments take effect at epoch start, too, I guess to avoid unfair penalties.

2. Your SLA is evaluated at epoch end for the time period for which you still had a commitment. This is a bit nasty because what if you have multiple different commitment levels during an epoch.

3. Yolo! Leave it and LPs can deal with it.

I'm leaning towards 1. Thoughts @TomMcL @davidsiska-vega @witgaw ?

Reply:
1 sounds nice and clean to me